cfgov/cfgov/settings/base.py

```diff
@@ -186,6 +186,8 @@
 
 ALLOWED_HOSTS = ['*']
 
+EXTERNAL_LINK_PATTERN = r'https?:\/\/(?:www\.)?(?![^\?]+gov)(?!(content\.)?localhost).*'
```

Comment:
This has been thoroughly tested, ya? I can't get it to pick up our domains in http://pythex.org/